### PR TITLE
Using AWS ecr prebuilt image to run tests

### PR DESCRIPTION
**Pull request by aalavandhan**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -42,6 +42,7 @@ USER fragments
 WORKDIR /home/fragments/dependencies
 RUN git clone https://github.com/trailofbits/echidna.git
 WORKDIR /home/fragments/dependencies/echidna
+RUN git reset --hard d93c226b2ad4ff884f33faa850d7d36556c12211 # Using stable version
```
Comment (ahnaguib) [inline Dockerfile]: Can we use a tag for this?

Comment (brandoniles) [inline Dockerfile]: Can we use a tag instead of a commit hash?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	@docker tag $(DOCKER_REPO):$(DOCKER_TAG) $(AWS_ECR_DOMAIN)/$(DOCKER_REPO):$(DOCKER_TAG_LATEST)
+	
+push: ecr-login tag
+	@docker push $(AWS_ECR_DOMAIN)/$(DOCKER_REPO):$(DOCKER_TAG)
+	@docker push $(AWS_ECR_DOMAIN)/$(DOCKER_REPO):$(DOCKER_TAG_LATEST)
+
+pull: ecr-login
+	@docker pull $(AWS_ECR_DOMAIN)/$(DOCKER_REPO):$(DOCKER_TAG_LATEST)
+
+exec: pull
```
Comment (ahnaguib) [inline Makefile]:
```suggestion
run: pull
```

**Review by brandoniles — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (nithinkrishna) [reply Dockerfile]: They've not tagged releases :/

**Pull request merged**